packages/graphics/src/styles/FillStyle.ts

```diff
@@ -0,0 +1,95 @@
+import { Texture } from '@pixi/core';
```

Comment:
Why didn't FillStyle show up as a git move?

Reply:
So, in github it look as move and change. Not as deleting,
![image](https://user-images.githubusercontent.com/12572222/73598218-8c2ce080-4546-11ea-89ec-ebac09623962.png)

Reply:
Weird, can’t trust GitHub UI